unit_tests/test_views.py

```diff
@@ -34,6 +34,7 @@ def test_about(self):
         self.assertContains(response, '<p>footnote text') #make sure that footnote was rendered
 
     def test_links(self):
+        models.Static.objects.create(title='Links', text='### <h2>Links</h2>')
```

Comment:
Here you could create the text as pure markdown (no html elements), and then the assertContains line would verify that the markdown is rendered. So this line might be something like:
models.Static.objects.create(title='Links', text='## Links')